### PR TITLE
Issue 241: NEB Db Page and Selection Modal Updates

### PR DESCRIPTION
**Pull request by rmroot**
connects #241 

Adds static page with list of NEBs in tool. Filterable by KPM and KPI
Use page in NEB select modal. Can search/filter by KPM and KPI

### CODE REVIEW

File: src/app/nebs-database/nebs-database-table/nebs-database-table.component.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                        <span class="inner-span" [innerHTML]="kpm.label"></span>
+                    </div>
+                </div>
+            </option>
+        </select>
+    </div>
+</div>
+<p class="text-right fw-light">
+    &ast; Indicates KPI/KPM currently tracked by this company.
+</p>
```
Comment (RLiNREL) [inline src/app/nebs-database/nebs-database-table/nebs-database-table.component.html:56]: perhaps, hide this when the user open the neb database from the homepage

File: src/app/nebs-database/nebs-database-table/nebs-database-table.component.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+<hr>
+@let tableList = nebOptions | nebOptionsFilter: orderByDir: kpiValue: kpmValue: keyPerformanceMetricOptions:
```
Comment (RLiNREL) [inline src/app/nebs-database/nebs-database-table/nebs-database-table.component.html:58]: that's a new feature 😊

File: src/app/nebs-database/nebs-database-table/nebs-database-table.component.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+nebSearchStr;
+<table class="table table-bordered table-hover table-sm">
+    <thead>
+        <tr class="clickable">
+            <th *ngIf="inAddModal">
+
+            </th>
+            <th (click)="toggleOrderBy()">
+                <fa-icon [icon]="faChevronDown" *ngIf="orderByDir == 'asc'"></fa-icon>
```
Comment (RLiNREL) [inline src/app/nebs-database/nebs-database-table/nebs-database-table.component.html:67]: Feel the chevron is not obvious. Perhaps, add some text to remind user there is a sorting feature, just a suggestion (no changes needed)

File: src/app/nebs-database/nebs-database-table/nebs-database-table.component.html (hunk 1 above)
Comment (rmroot) [reply src/app/nebs-database/nebs-database-table/nebs-database-table.component.html:56]: good catch. fixed

**Review by RLiNREL — APPROVED** [review]

**Pull request merged**